docs/rust-client/custom_note_how_to.md

```diff
@@ -0,0 +1,394 @@
+    let (account, seed) = builder.build().unwrap();
+    client
+        .add_account(
+            &account,
+            Some(seed),
+            &AuthSecretKey::RpoFalcon512(key_pair),
+            false,
+        )
+        .await?;
+    Ok(account)
+}
+
+#[tokio::main]
+async fn main() -> Result<(), ClientError> {
+    // Initialize client
+    let mut client = initialize_client().await?;
+    println!("Client initialized successfully.");
+
+    // Fetch latest block from node
+    let sync_summary = client.sync_state().await.unwrap();
+    println!("Latest block: {}", sync_summary.block_num);
+
+    //------------------------------------------------------------
+    // STEP 1: Create two basic accounts (Alice and Bob)
+    //------------------------------------------------------------
+    println!("\n[STEP 1] Creating new accounts");
+    let alice_account = create_basic_account(&mut client).await?;
+    println!("Alice's account ID: {:?}", alice_account.id().to_hex());
+    let bob_account = create_basic_account(&mut client).await?;
+    println!("Bob's account ID: {:?}", bob_account.id().to_hex());
+
+    //------------------------------------------------------------
+    // STEP 2: Mint and consume tokens for Alice with Ephemeral P2ID
+    //------------------------------------------------------------
+    println!("\n[STEP 2] Mint tokens with Ephemeral P2ID");
+
+0xf42ab06ffd227c2000005b2767bc5e    let faucet_id = AccountId::from_hex("0xf42ab06ffd227c2000005b2767bc5e").unwrap();
+    let amount: u64 = 100;
+    let fungible_asset_mint_amount = FungibleAsset::new(faucet_id, amount).unwrap();
+
+    let transaction_request = TransactionRequestBuilder::mint_fungible_asset(
+        fungible_asset_mint_amount.clone(),
+        alice_account.id(),
+        NoteType::Public,
+        client.rng(),
+    )
+    .unwrap()
+    .build();
+
+    let tx_execution_result = client
+        .new_transaction(faucet_id, transaction_request)
+        .await?;
+    client
+        .submit_transaction(tx_execution_result.clone())
+        .await?;
+
+    // The minted fungible asset is public so output is a `Full` note type
+    let p2id_note: Note =
+        if let OutputNote::Full(note) = tx_execution_result.created_notes().get_note(0) {
+            note.clone()
+        } else {
+            panic!("Expected Full note type");
+        };
+
+    let transaction_request = TransactionRequestBuilder::new()
+        .with_unauthenticated_input_notes([(p2id_note, None)])
+        .build();
+
+    let tx_execution_result = client
+        .new_transaction(alice_account.id(), transaction_request)
+        .await?;
+    client.submit_transaction(tx_execution_result).await?;
+    client.sync_state().await?;
+
+    // -------------------------------------------------------------------------
+    // STEP 3: Hash Secret Number and Build Note
+    // -------------------------------------------------------------------------
+    println!("\n[STEP 3] Create note");
```

Comment:
Can you say here "Create custom note" to make is super clear that this is what the tutorial is about